extra/constant.js

```diff
@@ -0,0 +1,27 @@
+import validateEmail from "./utils/helper";
+
+const permissions = {
```

Comment:
Please define string values as constants in this fle and use the variable rather than repeating the same string.